### PR TITLE
Line edits for Aggregate docs

### PR DESCRIPTION
**Pull request by mish24**
addresses #224 

## What is included in this PR?
* Minor line edits
* Sphinx specific syntax at required places,

### CODE REVIEW

File: aggregate-guide.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ODK Aggregate
 ******************
 
-`ODK Aggregate` provides a ready-to-deploy server and data repository to:-
+`ODK Aggregate` provides a ready-to-deploy server and data repository to :-
```
Comment (adammichaelwood) [inline aggregate-guide.rst]: It is not standard to put a space before colons.

I *would* like to see the hyphens after commas removed, as that is non-standard.

**Comment by adammichaelwood** [comment]
The merge conflicts should be easy to resolve here in GH's editor. Let me know if you can't figure it out.

**Comment by mish24** [comment]
@adammichaelwood Never tried that. I will use this `Resolve conflicts` and try to solve it.

**Comment by adammichaelwood** [comment]
Closing due to inactivity.
You can reopen or file a new PR.

**Pull request closed without merging**